packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -455,9 +455,11 @@ export class ProjectModel {
     }
 
     async delete(projectUuid: string): Promise<void> {
-        await this.database('projects')
-            .where('project_uuid', projectUuid)
-            .delete();
+        await this.database.transaction(async (trx) => {
+            await this.database('projects')
```

Comment:
```suggestion
            await trx('projects')
```

Reply:
yes, of course, good catch, I didn't notice